app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> str:
+    file_1, file_2 = command.split()[1], command.split()[2]
+    with open(file_1, "r") as file_in, open(file_2, "w") as file_out:
+        if file_in == file_out:
+            pass
```

Comment:
This is need to check before open because you do additional actions

Reply:
sorry, I didn't get what you mean.